### DISCUSSION TITLE
Welcome to ngrinder Discussions!

### DISCUSSION THREAD

**Discussion by junoyoon** [naver/ngrinder#699] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WANGJIANGA** [comment]


**Reply by donggyu04** [reply]
@WANGJIANGA 

You can confirm multi TPS in `Detail Report` page.

<img src="https://user-images.githubusercontent.com/14273601/102862567-a03a4300-4474-11eb-9d10-3da4d586b2a3.png" width="990" height="350" />

**Reply by WANGJIANGA** [reply]
Thank you for reply。There is only one TPS on the detailed report page

**Reply by donggyu04** [reply]
@WANGJIANGA 

You must define multiple `GTest` please check again.

```
public static GTest test1
public static GTest test2
public static GTest test3

...

test1 = new GTest(1, "Description-1")
test2 = new GTest(2, "Description-2")
test3 = new GTest(3, "Description-3")

...

test1.record(this, "Input your test function name-1")
test2.record(this, "Input your test function name-2")
test3.record(this, "Input your test function name-3")
```

**Reply by junoyoon** [reply]
Please do not reply in this. Please use separate threads.

**Reply by WANGJIANGA** [reply]
I'm sure my script is all right。
This problem happens occasionally.
my ngrinder version is 3.4.1

**Comment by junoyoon** [comment, marked as answer]
PLEASE DO NOT ADD COMMENT HERE. 
USE DIFFERENT THREAD
